packages/quantic/cypress/e2e/default-2/search-box/search-box.cypress.ts

```diff
@@ -0,0 +1,738 @@
+              Actions.focusSearchBox(textarea);
+              cy.wait(InterceptAliases.QuerySuggestions);
+              Expect.displaySuggestionList(true);
+              Expect.numberOfQuerySuggestions(exampleQuerySuggestions.length);
+              Expect.querySuggestionsEquals(exampleQuerySuggestions);
+              const clickedSuggestionIndex = 0;
+              Actions.clickQuerySuggestion(clickedSuggestionIndex);
+              Expect.searchWithQuery(
+                exampleQuerySuggestions[clickedSuggestionIndex],
+                {
+                  LSkey: recentQueriesLSKey,
+                  queries: [exampleQuerySuggestions[clickedSuggestionIndex]],
+                }
+              );
+            });
+          });
+        });
+
+        describe('when using the keyboard to select a recent query', () => {
+          const exampleRecentQueries = ['foo', 'bar'];
+
+          beforeEach(() => {
+            setRecentQueriesInLocalStorage(exampleRecentQueries);
+            visitSearchBox({...defaultOptions, textarea});
+            mockQuerySuggestions(exampleQuerySuggestions);
+          });
+
+          it('should properly navigate and select the right recent query', () => {
+            const expectedQuerySuggestions = [
+              ...exampleRecentQueries,
+              ...exampleQuerySuggestions,
+            ];
+            scope('when loading standalone search box', () => {
+              Expect.displaySearchBoxInput(true, textarea);
+              Expect.displaySearchButton(true);
+            });
+
+            scope('when focusing on the search box input', () => {
+              Actions.focusSearchBox(textarea);
+              cy.wait(InterceptAliases.QuerySuggestions);
+
+              Expect.displaySuggestionList(true);
+              Expect.displayClearRecentQueriesButton(true);
+              Expect.numberOfQuerySuggestions(expectedQuerySuggestions.length);
+              Expect.querySuggestionsEquals(expectedQuerySuggestions);
+            });
+
+            scope('when selecting a recent query with the keyboard', () => {
+              // First arrow down press to go to the clear recent queries option.
+              Actions.pressDownArrowOnSearchBox(textarea);
+              // Second arrow down press to go to the first recent queries option.
+              Actions.pressDownArrowOnSearchBox(textarea);
+              Actions.pressEnterOnSearchBox(textarea);
+              const clickedSuggestionIndex = 0;
+              Expect.searchWithQuery(
+                exampleRecentQueries[clickedSuggestionIndex],
+                {
+                  LSkey: recentQueriesLSKey,
+                  queries: exampleRecentQueries,
+                }
+              );
+              Expect.logClickRecentQueries(
+                exampleRecentQueries[clickedSuggestionIndex]
+              );
+            });
+          });
+        });
+
+        describe('when using the keyboard to select a query suggestion', () => {
```

Comment:
This test should test that when selecting a query suggestion it will execute that search and that's all?

Reply:
yes, just using the the keyboard.